spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb

```diff
@@ -163,13 +163,13 @@ class PodsProjectGenerator
                 @project = Project.new(config.sandbox.project_path)
                 config.sandbox.project = @project
 
-                @coconut_spec = fixture_spec('coconut-lib/CoconutLib.podspec')
+                @watermelon_spec = fixture_spec('watermelon-lib/WatermelonLib.podspec')
```

Comment:
re-wrote this to use `WatermelonLib` fixture which has two test specs and helps us test test spec split.